Makefile

```diff
@@ -605,23 +609,35 @@ bazel-build-cli:
 	bazel build //cmd/kops/...
 
 # Not working yet, but we can hope
-#.PHONY: bazel-crossbuild-kops
-#bazel-crossbuild-kops:
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/kops/...
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/kops/...
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:windows_amd64 //cmd/kops/...
-#
-#.PHONY: bazel-crossbuild-nodeup
-#bazel-crossbuild-nodeup:
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/nodeup/...
+.PHONY: bazel-crossbuild-kops
+bazel-crossbuild-kops:
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/kops/...
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/kops/...
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:windows_amd64 //cmd/kops/...
+
+.PHONY: bazel-crossbuild-nodeup
+bazel-crossbuild-nodeup:
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/nodeup/...
+
+.PHONY: bazel-crossbuild-protokube
+bazel-crossbuild-protokube:
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //protokube/...
 
-#.PHONY: bazel-crossbuild-protokube
-#bazel-crossbuild-protokube:
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //protokube/...
+.PHONY: bazel-crossbuild-dns-controller
+bazel-crossbuild-dns-controller:
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //dns-controller/...
 
-#.PHONY: bazel-crossbuild-dns-controller
-#bazel-crossbuild-dns-controller:
-#	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //dns-controller/...
+.PHONY: bazel-crossbuild-dns-controller-image
+bazel-crossbuild-dns-controller-image:
+	bazel build --experimental_platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //images:dns-controller.tar
+
+.PHONY: bazel-crossbuild-protokube-image
+bazel-crossbuild-protokube-image:
```

Comment:
Got a odd warning here:
```
WARNING: /private/var/tmp/_bazel_clove/8fb6d18c232e3eae2b829c764cbc089c/external/distroless_rules/WORKSPACE:1: Workspace name in 
/private/var/tmp/_bazel_clove/8fb6d18c232e3eae2b829c764cbc089c/external/distroless_rules/WORKSPACE (@distroless) does not match the name given in the repository's definition (@distroless_rules); this will cause a build error in future versions
```